ztp/gitops-subscriptions/argocd/example/acmpolicygenerator/acm-group-du-3node-validator-ranGen.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: PolicyGenerator
+metadata:
+  name: group-du-3node-validator-latest
+placementBindingDefaults:
+  name: group-du-3node-placement-binding
+policyDefaults:
+  # categories: []
+  #controls:
+  #  - PR.DS-1 Data-at-rest
+  namespace: ztp-group
+  # Use an existing placement rule so that placement bindings can be consolidated
+  placement:
+    labelSelector:
+      matchExpressions:
+      - key: group-du-3node
+        operator: Exists
+      - key: du-profile
+        operator: In
+        values: ["latest"]
+      - key: ztp-done
+        operator: DoesNotExist
+  remediationAction: inform
+  severity: low
+  # standards: []
+  namespaceSelector:
+    exclude:
+      - kube-*
+    include:
+      - '*'
+  evaluationInterval:
+    compliant: 5s
```

Comment:
There is a comment in the original PGT that indicates this low value is only valid (acceptable) if the binding rules disconnect the policy based on ztp-done. Otherwise we end up with high CPU use at steady state.